desktop/apps/consign/routes.js

```diff
@@ -53,13 +53,15 @@ export const submissionFlowWithId = async (req, res, next) => {
 
 export const submissionFlowWithFetch = async (req, res, next) => {
   try {
-    const token = await fetchToken(req.user.get('accessToken'))
-    const submission = await request
-                              .get(`${res.locals.sd.CONVECTION_APP_URL}/api/submissions/${req.params.id}`)
-                              .set('Authorization', `Bearer ${token}`)
-    const { artist: { name } } = await metaphysics({ query: ArtistQuery(submission.body.artist_id), req })
-    res.locals.sd.SUBMISSION = submission.body
-    res.locals.sd.SUBMISSION_ARTIST_NAME = name
+    if (req.user) {
```

Comment:
I'd seen this error in sentry but just caught it myself-- if you try to access one of the interim steps but are logged out. We may want to improve this flow further, but for now the only way to discover these paths is by going through the submission flow itself (i.e. we don't link to it), so it's pretty unlikely you'll be logged out mid-flow.